zxinglight/__init__.py

```diff
@@ -80,7 +81,7 @@ def read_codes(image, barcode_type=BarcodeType.NONE, try_harder=False, hybrid=Fa
         multi (bool): Search for multiple barcodes in a single image.
 
     Returns:
-        A list of barcode values.
+        A list [(code, position, type), ...] containing each barcode found.
```

Comment:
I'd be great if we returned [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple)s here instead of regular tuples. Working with long/nested tuples can quickly become awkward.